javascript/ql/experimental/adaptivethreatmodeling/test/qlpack.yml

```diff
@@ -0,0 +1,5 @@
+name: codeql/javascript-experimental-atm-tests
+version: 0.0.0
```

Comment:
If this pack is not going to be published, best to remove the version (prevents accidental publishing).

```suggestion
```